pkg/service/webhook/model.go

```diff
@@ -58,3 +72,40 @@ type GetSupportedNounsResponse struct {
 type GetSupportedVerbsResponse struct {
 	Verbs []Verb `json:"verbs,omitempty"`
 }
+
+func (cwr DeleteWebhookRequest) IsValid() bool {
+	fmt.Println(cwr)
+	if isValidNoun(cwr.Noun) && isValidVerb(cwr.Verb) && isValidURL(cwr.URL) {
+		return true
+	}
+	return false
+}
+
+func (cwr CreateWebhookRequest) IsValid() bool {
+	fmt.Println(cwr)
+	if isValidNoun(cwr.Noun) && isValidVerb(cwr.Verb) && isValidURL(cwr.URL) {
+		return true
+	}
+	return false
+}
+
+func isValidNoun(noun Noun) bool {
```

Comment:
instead this can be a receiver method such as

`func (n Noun) IsValid() bool { ... }`

same for below